style/main.css

```diff
@@ -91,47 +94,29 @@ nav {
 .nav-item a {
     text-decoration: none;
     color: rgb(70, 67, 67);
-    margin-right: 2rem;
+    /* margin-right: 2rem;  */
 }
 
-.nav-item a:hover {
+.nav-item a:hover, 
+a:hover {
     color: lightseagreen;
+    cursor: pointer;
 }
 
-.nav-list .logo-img{
-    width: 5rem;
-    padding: 0.75rem;
+.nav-list ul {
+    padding: 3rem;
 }
 
-
-/* ------------ Main Content ------------- */
-
-.news-feed {
-/* background-color: aquamarine; */
-height: 100%;
-}
-
-.news-feed .img-container {
-    width: 31%;
+ul.nav-list {
+    padding-inline-end: 2.5rem;
 }
 
-.news-feed .img-container img {
-    width: 100%;
-    border-radius: 10px;
-}
-
-section.news-feed {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    gap: 1rem; 
+.nav-list .logo-img{
+    width: 5rem;
+    padding: 0.75rem;
 }
 
 /* -------- Advertisement --------- */
-
-/* .advertisement {
-    background-color: burlywood;
-} */
  
 .advertisement-card {
     width: 20vw;
```

Comment:
I see that you are using different types of units, but for consistency and adaptive webapp it is better try to use one kind of units of measure (of course sometime you will need and px or/and  rem/em6 but vw, vh, rem, em, px use at the same time isn't a good practice)

Reply:
Yes, you are absolutely right. I'll refactor css files.

Reply:
I think that it is also ready. I've changed "vw" with "%" also in some places I converted "px" to "rem". I still have "px" for font-size, min-width, max-width, borders, but I suppose that it is fine. If you prefer I could change all "px" with "rem"?